dags/openshift_nightlies/manifest.yaml

```diff
@@ -15,6 +15,10 @@ versions:
   alias: future
   releaseStream: 4.10.0-0.nightly
   baremetalReleaseStream: latest-4.10
+- version: 4.11
+  alias: future
```

Comment:
Doesn't this leave us w/ 2 versions flagged as future? Will that break things?

Reply:
already done!